.dockerignore

```diff
@@ -0,0 +1,9 @@
+.git
+.direnv
+Dockerfile
+js
+test
+tmp
+bin
+input
+node_modules
```

Comment:
`node_modules` should be within the `js` dir so we shouldn't need to repeat it.

I'd also probably ignore the `rebuild` directory since the docs use that to test rebuilding into (or we could change the docs to point to `tmp/rebuild`)

We also likely want to ignore `bin` and `release` which gets created when binaries are built.

And we can probably ignore `docs` and `.vscode` as well.

Reply:
Should we maybe force the certs and key files from `development` to be ignore so we never accidentally deploy the service using dev keys.

Reply:
Word, ignored all of `development` except the nix stuff